subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -294,35 +292,24 @@ uint8_t ll_conn_iso_accept_timeout_set(uint16_t timeout)
 
 void ull_conn_iso_lll_cis_established(struct lll_conn_iso_stream *cis_lll)
 {
+	struct node_rx_pdu *node_rx;
 	struct ll_conn_iso_stream *cis =
 		ll_conn_iso_stream_get(cis_lll->handle);
 
 	if (cis->established) {
 		return;
 	}
 
-#if defined(CONFIG_BT_LL_SW_LLCP_LEGACY)
-	struct node_rx_conn_iso_estab *est;
-	struct node_rx_pdu *node_rx;
-
 	node_rx = ull_pdu_rx_alloc();
 	if (!node_rx) {
 		/* No node available - try again later */
 		return;
 	}
 
 	node_rx->hdr.type = NODE_RX_TYPE_CIS_ESTABLISHED;
+	node_rx->hdr.handle = cis->lll.acl_handle;
 
-	/* TODO: Send CIS_ESTABLISHED with status != 0 in error scenarios */
-	node_rx->hdr.handle = 0xFFFF;
-	node_rx->hdr.rx_ftr.param = cis;
-
-	est = (void *)node_rx->pdu;
-	est->status = 0;
-	est->cis_handle = cis_lll->handle;
-
-	ll_rx_put_sched(node_rx->hdr.link, node_rx);
-#endif /* !CONFIG_BT_LL_SW_LLCP_LEGACY */
+	ull_rx_put_sched(node_rx->hdr.link, node_rx);
```

Comment:
Something is not right here, this is not there for LLCP in the LHS

Reply:
removed